macroid-core/src/main/scala/macroid/MediaQueries.scala

```diff
@@ -85,10 +85,10 @@ private[macroid] trait SizeQueries extends MediaQueryEssentials {
 }
 
 private[macroid] trait MediaQueries
-  extends DensityQueries
-  with OrientationQueries
-  with DisplayUnits
-  with SizeQueries {
+    extends DensityQueries
```

Comment:
Not sure I agree with this formatting. Can this be configured to use a single indent?